src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/SQL/Migrations.cs

```diff
@@ -702,5 +702,29 @@ public int UpdateFrom3()
 
             return 4;
         }
+
+        // This code can be removed in a later version.
+        public int UpdateFrom4()
+        {
+            SchemaBuilder.AlterIndexTable<TimeFieldIndex>(table => table
+                .DropIndex("IDX_TimeFieldIndex_DocumentId_Time")
```

Comment:
This will fail for users of sql server pre 2016. 

The solution for them is to perform the migration manually and update their migration record.